cmd/dexctl/main.go

```diff
@@ -69,19 +75,41 @@ func wrapRun(run func(cmd *cobra.Command, args []string) int) func(cmd *cobra.Co
 	}
 }
 
-func getDriver() (drv driver) {
+func isDBURLPresent() bool {
 	var err error
 	switch {
 	case len(global.dbURL) > 0:
-		drv, err = newDBDriver(global.dbURL)
-	default:
-		err = errors.New("--db-url flag unset")
+		stdout("--db-url flag is deprecated. Use --base-url and --api-key")
+		return true
+	case len(global.baseURL) < 1:
+		err = errors.New("--base-url flag unset")
+	case len(global.apiKey) < 1:
+		err = errors.New("--api-key flag unset")
+
 	}
 
 	if err != nil {
 		stderr("Unable to configure dexctl driver: %v", err)
 		os.Exit(1)
 	}
 
-	return
+	return false
+}
+
+func getDBConnector() *dbConnector {
+	dbConnector, err := newDBConnector(global.dbURL)
+	if err != nil {
+		stderr("Unable to configure dexctl databse connector: %v", err)
+		os.Exit(1)
```

Comment:
Same comment as the DB comment. It's just more idiomatic Go to service these error rather than exiting.